tests/scripts/thread-cert/Cert_9_2_04_ActiveDataset.py

```diff
@@ -182,6 +188,96 @@ def test(self):
         for ipaddr in ipaddrs:
             self.assertTrue(self.nodes[LEADER].ping(ipaddr))
 
+    def verify(self, pv):
+        pkts = pv.pkts
+        pv.summary.show()
+
+        LEADER = pv.vars['LEADER']
+        COMMISSIONER = pv.vars['COMMISSIONER']
+
+        # Step 1: Ensure the topology is formed correctly
+        pkts.filter_wpan_src64(LEADER).filter_mle_cmd(MLE_CHILD_ID_RESPONSE).must_next()
+
+        # Step 2: Commissioner sends MGMT_ACTIVE_SET.req to Leader RLOC
+        pkts.filter_wpan_src64(COMMISSIONER).filter_coap_request(MGMT_ACTIVE_SET_URI).must_next().must_verify(
```

Comment:
Validate destination is Leader RLOC?

Reply:
sure, check Leader RLOC or ALOC in the latest commit